src/components/book/Book.js

```diff
@@ -1,9 +1,9 @@
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../../redux/features/books/booksSlice';
+import { deleteBooks } from '../../redux/features/books/booksSlice';
 import './Book.css';
 
 const Book = ({
-  itemId,
+  id,
   title,
   author,
   category,
```

Comment:
You did a brilliant job for sure. Add and remove functions working properly 👍  But let's use useEffect for fetching the books as requested on project requirements.
<img width="751" alt="Screenshot 2023-03-02 at 7 34 21 PM" src="https://user-images.githubusercontent.com/25887253/222493345-3ec9a8ec-ff1a-417a-a3c0-9f28b0e8b563.png">

Reply:
Hey @huseyinbiyik Thank you for your feedback I appreciate. You have pointed out that I should fetch using the useEffect. That is exactly what I did. If you go to my pages/books, which is where I passed in the Book component, you'll notice that I used useEffect to fetch the books from the API.
![HuseyinCR](https://user-images.githubusercontent.com/83401478/222534614-1321968a-cce7-42db-b1b4-c228de40c838.PNG)

It is after fetching that I pass the data to the book component as props. The point is that I fetch books from the bookstore API when the component has rendered using useEffect. Kindly take that into consideration.

Thank you.

Reply:
@Fene-87 ops, I am so sorry that I didn't notice that. Sometimes, it is happening. But one thing I am sure of is your code looks brilliant in general 👍